.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
+
+## Title Line Template: [Website] - [UI components]: [Brief description.]
+<!-- Use the title line as the title of your pull request, then delete these lines. 
+
+Website: Issues that impact standards.usa.gov look, feel or functionality.
+UI components: Issues that impact the look, feel or functionality of the standards themselves.
+
+-->
```

Comment:
If a user reviews the template file themselves, they can see the comments. Otherwise the comments would appear in the `textarea` on the PR and Issue pages. We could remove these instructions here and include them in the contributing guide.

When users are creating a PR or Issue the template will be used exactly and these notes shouldn't be rendered in the final preview / rendering of the markdown.